### ISSUE TITLE
Path to Custom Stylesheets

### ISSUE THREAD

**Issue by nagymusic** [Abjad/abjad#1159]
Hello,

I just installed the Abjad 3.1 update (Mac, Python 3.7, Lilypond 2.19.83). While the PDF files are still created automatically in the `.abjad output` folder,  I can't seem to figure out why `abjad.show(lilypond_file)` is not opening a PDF file with the music typeset to reflect the included stylesheet. It's interesting that when I manually open the PDF file found in the `output` folder, the file is properly compiled; and all the necessary stylesheets can be found in the `output` folder.

Also, the PDF file titles in the `output` folder now include a long string of letters/numbers, instead of 0001, 0002, etc. Not sure if this is new or I have compromised my settings with the new Abjad installation, or my LilyPond settings are not compatible?

The code below worked just fine in Abjad 3.0:
```
lilypond_file = abjad.LilyPondFile.new(
    music=score,
    includes=['default_.ily'],
    )

abjad.show(lilypond_file)
```
This time, it produces an error message:
```
GNU LilyPond 2.19.83
Changing working directory to: `/var/folders/5f/ssfq4vzn4xv7kym13hhgh4kw0000gn/T/tmp_2_xsxr_'
Processing `/var/folders/5f/ssfq4vzn4xv7kym13hhgh4kw0000gn/T/tmp_2_xsxr_/2020-01-07T08-31-06-865198-16de11a.ly'
Parsing...
/var/folders/5f/ssfq4vzn4xv7kym13hhgh4kw0000gn/T/tmp_2_xsxr_/2020-01-07T08-31-06-865198-16de11a.ly:4:10: error: cannot find file: `default_.ily'
(search path: `/var/folders/5f/ssfq4vzn4xv7kym13hhgh4kw0000gn/T/tmp_2_xsxr_:/Users/nagy/Dropbox/Create/ABJAD/Practice Examples/class:/Applications/LilyPond.app/Contents/Resources/share/lilypond/current/fonts/svg/:/Applications/LilyPond.app/Contents/Resources/share/lilypond/current/fonts/type1/:/Applications/LilyPond.app/Contents/Resources/share/lilypond/current/fonts/otf/:/Applications/LilyPond.app/Contents/Resources/share/lilypond/current/scm:/Applications/LilyPond.app/Contents/Resources/share/lilypond/current/ps:/Applications/LilyPond.app/Contents/Resources/share/lilypond/current/ly:')
\include 
         "default_.ily" %! abjad.LilyPondFile._get_formatted_includes()
Interpreting music...[8]
Preprocessing graphical objects...
Finding the ideal number of pages...
Fitting music on 1 page...
Drawing systems...
Layout output to `/var/folders/5f/ssfq4vzn4xv7kym13hhgh4kw0000gn/T//lilypond-FzQ7qm'...
Converting to `2020-01-07T08-31-06-865198-16de11a.pdf'...
Deleting `/var/folders/5f/ssfq4vzn4xv7kym13hhgh4kw0000gn/T//lilypond-FzQ7qm'...
fatal error: failed files: "/var/folders/5f/ssfq4vzn4xv7kym13hhgh4kw0000gn/T/tmp_2_xsxr_/2020-01-07T08-31-06-865198-16de11a.ly"
```

Thank you,
~ Zvony

**Comment by nagymusic** [comment]
Just a quick update: I was able to in part solve my issue with this:
```
lilypond_file = abjad.LilyPondFile.new(
    music=staff,
    includes=['/path_to_stylesheet(s)'],
    )

# opens the PDF file (.abjad/output):
abjad.show(lilypond_file)

# saves the PDF in the custom folder:
abjad.persist(lilypond_file).as_pdf("/path_to_custom_folder/custom_title.pdf")
```
I'm not sure if this is the best way to go around it, but it works. Thanks again!

**Comment by trevorbaca** [comment]
You found the right solution @nagymusic: use fully qualified paths.

And the change you're observing in the way that Abjad names "anonymous" output files is correct: Abjad 3.0 and earlier used 0000.pdf, 0001.pdf, etc, as an increasing sequence of 4-digit nonnegative integers.

Starting in Abjad 3.1 the old file-naming pattern has been replaced. Calling `abjad.show()` in Abjad 3.1 causes three related files to write to your `~/.abjad/output` directory: `.ly`, `.pdf`, `.log`. Each of the three files is named with a big-endian timestamp together with a bit of hexidecimal hash. Here's an example:
```
2020-03-03T14-53-40-567081-681bc8b.log
2020-03-03T14-53-40-567081-681bc8b.ly
2020-03-03T14-53-40-567081-681bc8b.pdf
```
The portion of the filename stem that reads `2020-03-03T14-53-40` tells you that I called `abjad.show()` on 3 March 2020 at 14:53 (and 40 seconds) in the afternoon. The remaining `567081-681bc8b` part of the filename stem is a hash that uniquely identifies the content of the file; note that the hash isn't designed to be human-readable.
